indico/modules/events/papers/templates/_contributions.html

```diff
@@ -3,11 +3,18 @@
 {% macro render_paper_section(contribution) %}
     {% set paper = contribution.paper %}
     {% set can_submit = contribution.can_submit_proceedings(session.user) %}
-    {% set show_paper = paper and paper.accepted_revision %}
```

Comment:
{% set show_paper = paper and paper.accepted_revision and not paper.event.cfp.hide_accepted %}

Wouldn't that be enough? cc @duartegalvao

Reply:
That would keep the paper hidden for managers/the submitter too, right? They can still access from the timeline I think, but this way would be more convenient

Reply:
Yes, it does keep it hidden, but I want to +1 this for easy access.

Reply:
OK, fine then ;)